docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,127 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. This includes creating a GitHub application, so {ProductName} can access users' pull requests, and configuring a smee channel, to listen for those PRs. The first procedure in this document explains how to perform these tasks. 
+
+Once build pipelines are enabled, users can create their applications in {ProductName}. The second procedure below describes how to create an application. 
+
+== Enabling build pipelines for your instance of {ProductName}:
+
+.Prerequisites:
+
+* Install link:https://github.com/konflux-ci/konflux-ci/tree/main[{ProductName}]
+* Read/write access to your {ProductName} instance’s repository
+
+.Procedure:
+
+. Create a GitHub app according to link:https://pipelinesascode.com/docs/install/github_apps/#manual-setup[these Pipelines as Code instructions].
+. Start a new link:https://smee.io/[smee] channel.
+. In the repository for your instance of {ProductName}, edit the file /smee/smee-client.yaml. Replace <smee-channel> with the webhook proxy URL from the previous step. 
+
+== Creating your applications and components with {ProductName}:
+
+.Prerequisites:
+
+* link:https://kubernetes.io/docs/tasks/tools/[kubectl] CLI tool
+* The following values from your {ProductName} GitHub application:
+** The private key
+** The application ID
+** The webhook secret
+* A GitHub repository with source code for your application 
+** It must have the same basic structure as our example repository
+* Read/write access to the repository for your instance of {ProductName} 
+
+.Procedure:
+
+. In the GitHub repository for your application, install your organization’s {ProductName} GitHub application.
+. In the repository for your instance of {ProductName}, you need to replace two `url` values in the relevant YAML file for your namespace. The path for this file is `/test/resources/demo-users/user/<namespace>/application-and-component.yaml`. Replace the two `url` values as follows:
```

Comment:
The procedure we're working on in konflux-ci repo is focusing on user2. We're not making sure user1 has valid configs at this point. If this document is about following that procedure, then it should just mention user2.

If it supposed to be generic, then the resources (e.g. application and component) cannot be used as-is from the files used for that procedure, but they can act as references.

Reply:
I think I responded to your feedback in the doc as best I can, given that we're documenting the planned downstream, not the current upstream. But I'm least certain about this part. 

Whenever you have the time (no worries if you want to leave it for tomorrow), please let me know if the latest revision addresses this concern. Or if it doesn't, how I can fix it.

Thanks!